lib/shopify-cli/tasks/ensure_env.rb

```diff
@@ -22,6 +22,9 @@ def call(ctx, regenerate: false, required: [:api_key, :secret])
       private
 
       def fetch_org
+        if Shopifolk.check && wants_to_run_against_shopify_org?
```

Comment:
This impacts the following commands:

* Connect
* Scripts - Enable
* Scripts - Disable
* Scripts - Connect
* Populate Resource